### ISSUE TITLE
Level 2 headings are not read in the Settings page

### ISSUE THREAD

**Issue by jinglouMSFT** [microsoft/AzureStorageExplorer#2730]
Build 20200318.10
Windows 10
NVDA & Narrator

The level 2 headings are not read in the Settings page.

1. Start SE
2. Click on the settings page
3. Select "Application" on the right side
4. Turn on either NVDA or Narrator
5. Set focus on "Transfers" using mouse or keyboard
6. Tab through the list

Observe:
Level 2 headings are not read:
- "AzCopy - Use these settings to tune the performance ..... ADLS Gen2 and Managed Disks." not read
- "Table Import/Export" - not read

This also repro on the "Application" page, such as "Appearance", "Regional Settings", "Sign-in" are not read.

**Comment by jinglouMSFT** [comment]
Nevermind. Tab doesn't read the whole document.  Needs to turn NVDA into reading document mode to have all contents read.  Use Alt-ctrl+shift plus navigation keys to read all content.
